src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -11,67 +11,100 @@ import askgrok from 'images/grok_screenshot-crop_ask-grok.webp'
 
 import nrqlconsole from 'images/grok_screenshot-full_nrql-console.webp'
 
-New Relic AI (NRAI) is designed to help engineers improve observability. We combined OpenAI's large language models (LLMs) with our unified telemetry data platform to create an AI assistant that deeply understands how both your system and New Relic function. New Relic AI helps you demystify the complexity of your stack, solve service outages, and improve your efficiency simply by letting you ask questions in plain language. 
+import aiNewRelicAiExplainThisError from 'images/ai_screenshot-crop_new-relic-ai-explain-this-error.webp'
 
-<Video
-  type="youtube"
-  id="XMSUVmNKVyM"
-/>
+import aiExplainThisErrorPopUp from 'images/ai_screenshot-crop_explain-this-error-pop-up.webp'
+
+import aiAskAiinErrorsInbox from 'images/ai_screenshot-crop_ask-ai-in-errors-inbox.webp'
+
+New Relic AI (NRAI) is designed to help every member of your team improve observability, not just engineers. We combined OpenAI's large language models (LLMs) with our unified telemetry data platform to create an AI assistant that deeply understands how both your system and New Relic function. New Relic AI helps you demystify the complexity of your stack, solve service outages, and improve your efficiency simply by letting you ask questions in plain language. 
 
 ## Ways to interact with New Relic AI [#work-with-NRAI]
 
 We help you get to the heart of your system by offering two quick and easy ways to ask questions: AI chat, and the NRQL console. 
```

Comment:
It seems like boiling the options down to two approaches might be too simplistic since you're offering four different experiences:

* Ask AI questions
* Get help creating NRQL queries
* Get help analyzing error logs
* Analyze stack traces in errors inbox

Maybe you could just skip the number of ways and let the headings show users how they might interact with New Relic AI?